packages/@lwc/compiler/src/__tests__/fixtures/expected-babel-compat.js

```diff
@@ -80,7 +80,7 @@ writable: true
 __setKey(obj, key, value);
 }
 return obj;
-}
+} // babel-plugin-check-es2015-constants
```

Comment:
i don't know why now this babel comment is added. do you have an idea @diervo @pmdartus ?

Reply:
This is probably a bug with comments in Babel, I wouldn't be worried about this. I already had to deal with those comments in the past.